JavaScript/JavaScriptSDK/Telemetry/PageView.ts

```diff
@@ -21,9 +22,10 @@ module Microsoft.ApplicationInsights.Telemetry {
         /**
          * Constructs a new instance of the PageEventTelemetry object
          */
-        constructor(name?: string, url?: string, durationMs?: number, properties?: any, measurements?: any) {
+        constructor(id?: string, name?: string, url?: string, durationMs?: number, properties?: any, measurements?: any) {
```

Comment:
this change may break customers who constructing pageView today. especially since every argument here is optional and some people may rely on position of argument, not name. + @OsvaldoRosado who did some API clean up recently. What would be your suggestion?

Reply:
I would not change the order of any positional arguments, especially because customers get automatically upgraded to the latest version. This new parameter should be added to the end instead. That is, of course, only if this is part of the public API. Do we expose this class to end users or do we only abstract it behind a track method?

In terms of API cleanup, we addressed this upgrade conflict issue by making all of our TelemetryClient functions take a dictionary of arguments instead of positional arguments, but I don't think we can make a similar change here without breaking everyone.

Reply:
put id as a last parameter